pdf viewer - annotations (#123)
DevExpress/dotnet-eud: interface-elements-for-desktop/articles/pdf-viewer.md

legacyId: 17451
---
# PDF Viewer

> This section describes the **PDF Viewer** capabilities.

Reviewer: "the **PDF Viewer** capabilities" There aren't PDF Viewer capabilities, there are the capabilities of the PDF Viewer or the PDF Viewer's capabilities. However, this does not explain what you mean when you say **capabilities**.

> The PDF Viewer is used to display PDF files and supports various document content such as text, images, vector graphics, etc.

Reviewer: ```suggestion
The PDF Viewer can display PDF files with different document content such as text, images, vector graphics, etc.
```

![pdf-viewer-control-windows-forms](../images/img19650.jpg)

> The following sections are available:

Reviewer: Why do you say that sections are available? Do you mean this topic consists of these sections?
